src/Components/Feed.js

```diff
@@ -1,16 +1,64 @@
 import React, { Component } from 'react';
 import {NavLink} from 'react-router-dom';
+import { connect } from 'react-redux';
+import { getAll } from "../redux/actions/CreatePost";
 
 class Feed extends Component {
+
+  componentDidMount() {
+    this.props.getAllPosts();
+  }
+
+  displayPosts() {
+    return this.props.posts.items.map(post => {
```

Comment:
why don't you implement mapStateToProps in next way:

const mapStateToProps = ({posts: {isLoading, items}}) => {
  return {
    posts: items,
    isLoading
  }
}